### ISSUE TITLE
Adding extra fuel to fuel.toml doesnt work

### ISSUE THREAD

**Issue by Generalsquirrel** [henkelmax/ultimate-car-mod#159]
The mod doesnt allows me to add extra fuel types (like the bio diesel of immersive engineering)

Extra added Fuel to be atleast accepted by the fuel pump and also to go into the car (preferably also being accepted by the generator)

**Versions**
- Minecraft version 1.19.2
- Forge version 
- Mod version 1.0.0 & 1.0.4

**Other mods**
Immersive engineering,
Ultimate planes,
Jei,
Journey maps,
Create with a bunch of addobs (like factory must grow)

**Screenshots (Optional)**
Screenshots of the issue.

**Comment by henkelmax** [comment]
![image](https://github.com/henkelmax/ultimate-car-mod/assets/13237524/d9196d43-7954-4ea1-a0cb-30e9555152e4)
![image](https://github.com/henkelmax/ultimate-car-mod/assets/13237524/b662e29a-b279-443e-ba67-8632db338c6f)
![image](https://github.com/henkelmax/ultimate-car-mod/assets/13237524/be770433-c783-4c76-a121-04da1eb01bd2)

Works perfectly fine

**Comment by Generalsquirrel** [comment]
Oh then i have must made an error trying to get it to work my fault

**Comment by UshanB** [comment]
I still have the same issue - tried what was said earlier but game kept crashing. #minecraft:lava works but when I try to use any other mod fluids it crashes. I tried many ways such as using #createdieselgenerators:biodiesel or #forge:createdieselgenerators/biodiesel or #forge:biodiesel. Even using Immersive Engineering mod fuels seems to crash too. I did add the fuels to the server files as shown in the earlier snippet but still doesn't work
